gallery/gallery/lib/demos/material/sliders_demo.dart

```diff
@@ -0,0 +1,504 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+enum SlidersDemoType {
+  sliders,
+  rangeSliders,
+  customSliders,
+}
+
+class SlidersDemo extends StatelessWidget {
+  const SlidersDemo({Key key, this.type}) : super(key: key);
+
+  final SlidersDemoType type;
+
+  String _title(BuildContext context) {
+    switch (type) {
+      case SlidersDemoType.sliders:
+        return GalleryLocalizations.of(context).demoSlidersTitle;
+      case SlidersDemoType.rangeSliders:
+        return GalleryLocalizations.of(context).demoRangeSlidersTitle;
+      case SlidersDemoType.customSliders:
+        return GalleryLocalizations.of(context).demoCustomSlidersTitle;
+    }
+    return '';
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    Widget sliders;
+    switch (type) {
+      case SlidersDemoType.sliders:
+        sliders = _Sliders();
+        break;
+      case SlidersDemoType.rangeSliders:
+        sliders = _RangeSliders();
+        break;
+      case SlidersDemoType.customSliders:
+        sliders = _CustomSliders();
+    }
+    return Scaffold(
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        title: Text(_title(context)),
+      ),
+      body: Theme(
+        // Set platform to display material sliders only.
+        data: Theme.of(context).copyWith(platform: TargetPlatform.android),
```

Comment:
Why is this necessary? Oh I see it uses `Slider.adaptive`. I'd remove this `Theme` override and replace the `Slider.adaptive` calls with regular `Slider`

Reply:
Good call.